File: ljh/javascript/level1/test/IsIntegerSquareRoot.test.js
import IsIntegerSquareRoot from '../IsIntegerSquareRoot';
test(`정수 제곱근 판별`, () =>{
    expect(new IsIntegerSquareRoot().solution(121)).toBe(144);
}) 
test(`정수 제곱근 판별`, () =>{
    expect(new IsIntegerSquareRoot().solution(3)).toBe(-1);
}) 

File: ljh/java/src/level1/test/IsIntegerSquareRootTest.java
package level1.test;

import org.junit.Before;
import org.junit.Test;

import level1.IsIntegerSquareRoot;

public class IsIntegerSquareRootTest {
	private long[] n;
	private	long[] answers;
	private IsIntegerSquareRoot solution;
	
	@Before
	public void setUp() throws Exception {
		n = new long[] {121, 3};
		answers = new long[] {144, -1};
		solution = new IsIntegerSquareRoot();
	}
	
	@Test
	public void test() {
		for(int i=0; i<answers.length; i++) {
			assert(solution.solution(n[i]) == answers[i]);
		}
	}
}

